### PR TITLE
Depreciate Metrics CI workflows

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
Currently the master branch of CORE-V-VERIF has a set of Metrics-CI workflows that are no longer used.  This PR removes them.

@silabs-robin, I have asked you to review based on the discussion with @eroom1966 on MatterMost ([link](https://mattermost.openhwgroup.org/all-users/pl/5yexs4fuhjbyzdo5uwa7m1qeyr)).

@cathales, I have asked you to review as one of these workflows was written by you. I _believe_ it is no longer used, but would like your input before removing it.

### CODE REVIEW

File: .github/workflows/dashboard-done.yml
Comment (cathales) [inline .github/workflows/dashboard-done.yml:1]: This file makes it possible for https://riscv-ci.pages.thales-invia.fr/dashboard/dashboard_core-v-verif_0.html to automatically post a comment in CVA6 related PRs. For instance https://github.com/openhwgroup/core-v-verif/pull/2167#issuecomment-1703274656

Please don’t remove this file before https://github.com/openhwgroup/cva6/pull/1351 is merged. Then, I’ll probably delete it while removing `cva6/` directory from `core-v-verif` repository.

Comment (MikeOpenHWGroup) [reply .github/workflows/dashboard-done.yml:1]: Hi @cathales.  Perhaps I can close this PR and open a new one that deletes the other YAMLs, but not this one.  Would that work for you?

Comment (cathales) [reply .github/workflows/dashboard-done.yml:1]: @MikeOpenHWGroup I would like to update my answer with another solution: if we get https://github.com/openhwgroup/core-v-verif/pull/2168 merged today, we can proceed with updating and merging https://github.com/openhwgroup/cva6/pull/1351 and then merge this (#2174) PR as is.

Comment (cathales) [reply .github/workflows/dashboard-done.yml:1]: It is okay now, we do not need this file anymore. Thanks for your patience

**Comment by MikeOpenHWGroup** [comment]
Hi @silabs-robin, can you have a quick look?  Now that Côme's concerns are addressed, it would be good to get your eyes on this.

**Review by silabs-robin — APPROVED** [review]
Approving based on:
The removal of these files should not be a problem for us.
The author of one of the workflows has confirmed it is okay.
If anyone is negatively impacted, it is trivial to revert.

**Pull request merged**